examples/trigger.yaml

```diff
@@ -0,0 +1,27 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: network-delay
+  namespace: chaos-demo
+  annotations:
+    chaos.datadoghq.com/environment: "lima"
+spec:
+  level: pod
+  selector:
+    app: demo-curl
+  count: 100%
+  staticTargeting: true
+  trigger:
```

Comment:
@luphaz How do you feel about this structure? Let's us control injection and pod creation separately.

Reply:
Ok, so here inject means when the disruption is effectively applied, that relates to what we have today in the status as `Injected`?

And `pods` is, when will I create the pods for the given disruption?

Inject feels OK, however `pods` I'm not sure to what it relates to 🤔 

It's seems OK for me to split both however as they are not the same thing at all, and it make sense IMO to clearly isolate them.

Maybe `create` instead of pods? `deploy`, `run`?

Reply:
The names are definitely insufficient, I always put that last. But seems like you're good with splitting them like this, so I'll move forward with implementing and we can debate names as a team